swarm/README.md

```diff
@@ -0,0 +1,53 @@
+Mini-broker Swarm deployment for isolated containerized filters
+--------------------------------------------------------------
+
+These are Swarm deployment scripts for a version of the
+mini-broker running a one-container-per-filter setup.
+This design is updated from the design described in DMTN-081
+[https://dmtn-028.lsst.io](https://dmtn-028.lsst.io), which
+runs multiple filters in the same container.
+To deploy a prototype of the full mini-broker with 100 consumers
+(end users) and 100 individual filters each in their own container,
+run the `deploy_all.sh` script.
+This will deploy the following setup on a total of 12 nodes (one manager and
+11 worker nodes):
```

Comment:
What is a node, and what resources does it have (if that's even a valid question)?

Reply:
Oops, yes, I put in a description of the AWS instance types I used, to clarify.